### PR TITLE
Change description column

### PR DESCRIPTION
**Pull request by KevinHoughton**
Change the column from string to text to prevent errors about limitations if you exceed the amount of characters (191).

### CODE REVIEW

**Comment by Gummibeer** [comment]
I like the idea but would prefer it in a new migration that updates the column. Otherwise it will just effect new installations and don't update existing ones.

**Comment by KevinHoughton** [comment]
@Gummibeer I agree with you, I've updated the PR.

**Comment by Gummibeer** [comment]
And cause this updates/modifies a column this package will now require `doctrine/dbal` in composer.json
https://laravel.com/docs/5.5/migrations#modifying-columns

File: composer.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     ],
     "require": {
         "php" : "^7.0",
+        "doctrine/dbal": "~2.6",
```
Comment (Gummibeer) [inline composer.json]: `^2.5` will solve the composer problem on travis with php7.0

**Comment by freekmurze** [comment]
I've changed the migration in this package. `description` now uses a `text` column. 

To fix an existing installation in an app, just change the column type with a migration in your own project.

**Pull request closed without merging**